source/iface/context.rst

```diff
@@ -135,9 +136,12 @@ Returns information about the context as determined by ``param``. See
 ::
 
   enum class context : int {
```

Comment:
Maybe we should add here something like this:
namespace sycl {
namespace info {
<context enum definition>
}
}

Reply:
I was using existing template : 
_enum class context : int {_

Let's proceed like this and wait if Khronos open issue similar https://github.com/KhronosGroup/SYCL_Reference/issues/72